src/main/java/com/featureprobe/sdk/server/NumberMatcher.java

```diff
@@ -0,0 +1,10 @@
+package com.featureprobe.sdk.server;
+
+import java.util.List;
+
+@FunctionalInterface
+public interface NumberMatcher {
+
+    boolean match(float customValue, List<String> objects);
```

Comment:
Use Double to receive Number 
```suggestion
    boolean match(double customValue, List<String> objects);
```